knative-operator/deploy/resources/knativekafka/controller/eventing-kafka-controller.yaml

```diff
@@ -1648,7 +1648,6 @@ webhooks:
     reinvocationPolicy: IfNeeded
     matchPolicy: Equivalent
     namespaceSelector:
-      matchExpressions: [ ]
```

Comment:
upstream removes this here: https://github.com/knative-sandbox/eventing-kafka-broker/pull/2715